frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistScene.tsx

```diff
@@ -134,6 +134,7 @@ export function SessionRecordingsPlaylistScene(): JSX.Element {
             {playlist.short_id && pinnedRecordings !== null ? (
                 <div className="SessionRecordingPlaylistHeightWrapper">
                     <SessionRecordingsPlaylist
+                        logicKey={playlist.short_id}
```

Comment:
This seemed the right thing to do. Am I missing anything why the playlist component was not aware of it's id?

Reply:
@daibhin just in case you know the answer here having been in playlist land recently

otherwise I think it's just "wasn't needed yet"